http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -103,11 +106,12 @@ object appQuery extends TableQuery(new AppTable(_)) {
       app.nodepoolId,
       app.appType,
       app.appName,
+      app.appAccessScope,
       app.workspaceId,
       app.status,
       app.chart,
       app.release,
-      app.samResourceId,
+      AppSamResourceId(app.samResourceId.resourceId, app.appAccessScope),
```

Comment:
Ah re my comment in `LeoProfile`, I guess it works because the `AppSamResourceId` is copied here. 👍